base/models/utils.go

```diff
@@ -3,10 +3,13 @@ package models
 import (
 	"gorm.io/driver/postgres"
 	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
 )
 
-// GetGormConnection creates gorm database śtruct
+// GetGormConnection creates gorm database struct
 func GetGormConnection(dsn string) (*gorm.DB, error) {
-	db, err := gorm.Open(postgres.Open(dsn))
+	db, err := gorm.Open(postgres.Open(dsn), &gorm.Config{
+		Logger: logger.Default.LogMode(logger.Error),
```

Comment:
What is this logging when added? Why logger.Error?

Reply:
That was done to debug the queries, but then I have leave that in Error mode, as if I got it correctly it is a default mode.

Reply:
Ok, thanks